test/unit/test-action.js

```diff
@@ -318,47 +319,37 @@ describe('ActionService parseAction', () => {
     expect(a.args['key1']).to.deep.equal({expression: 'foo.bar'});
   });
 
-  it('should return null for undefined references in dereferenced arg', () => {
-    const a = parseAction('e:t.m(key1=foo.bar)');
-    expect(dereferenceExprsInArgs(a.args, null)).to.deep.equal({key1: null});
-    expect(dereferenceExprsInArgs(a.args, {})).to.deep.equal({key1: null});
-    expect(dereferenceExprsInArgs(a.args, {foo: null}))
-        .to.deep.equal({key1: null});
-  });
-
-  it('should return null for non-primitives in dereferenced args', () => {
-    const a = parseAction('e:t.m(key1=foo.bar)');
-    expect(dereferenceExprsInArgs(a.args, {foo: {bar: undefined}}))
-        .to.deep.equal({key1: null});
-    expect(dereferenceExprsInArgs(a.args, {foo: {bar: {}}}))
-        .to.deep.equal({key1: null});
-    expect(dereferenceExprsInArgs(a.args, {foo: {bar: []}}))
-        .to.deep.equal({key1: null});
-    expect(dereferenceExprsInArgs(a.args, {foo: {bar: () => {}}}))
-        .to.deep.equal({key1: null});
```

Comment:
Why remove these tests?

Reply:
i thought that the tests added no value but then again we need to ensure old behavior as clients will rely on this.